src/main/java/it/gov/pagopa/payhub/activities/dto/classifications/IufClassificationActivityResult.java

```diff
@@ -0,0 +1,17 @@
+package it.gov.pagopa.payhub.activities.dto.classifications;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class IufClassificationActivityResult {
+    private List<ClassifyResultDTO> classifyResultDTOS;
```

Comment:
As discussed on Slack, we could just reason on IUV, thus we could avoid a separate DTO
```suggestion
    private Set<String> iuvList;
```

Reply:
to discuss